docker/server/Dockerfile

```diff
@@ -1,3 +1,17 @@
+#
+# conductor:server - Netflix conductor server
+#
```

Comment:
Please update the comment to explain this stage.

Reply:
How about comment like this:
> 0. builder stage
> 1. bin stage

Reply:
Sounds good